src/pages/lib-v1/project-setup/create-application/q/platform/[platform].mdx

```diff
@@ -29,37 +29,247 @@ import flutter2 from '/src/fragments/lib-v1/project-setup/native_common/create-a
+import versionRangeCallout from '/src/fragments/lib-v1/ssr/nextjs/version-range-callout.mdx';
+
+<Fragments fragments={{ js: versionRangeCallout }} />
+
+To set up the project, you will need to create a new Next.js app with the
+[`create-next-app`](https://nextjs.org/docs/api-reference/create-next-app) tool.
+You'll then add Amplify and initialize a new project.
+
+Run the following command and following the instructions to create a Next.js app.
+
+```shellscript
+npx create-next-app@latest next-amplified
+```
+
+Then run the following command to enter the root of your Next.js app.
+
+```shellscript
+cd next-amplified
+```
+
+You can now run the app in development mode by using the following command:
+
+```bash
+npm run dev
+```
 
 </Block>
 
 <Block name="Angular">
 
-<Fragments fragments={{ js: js0 }} />
+Use the [Angular CLI](https://github.com/angular/angular-cli) to bootstrap a new Angular app:
+
+```bash
+npx -p @angular/cli ng new amplify-app
+
+? Would you like to add Angular routing? Y
+? Which stylesheet format would you like to use? (your preferred stylesheet provider)
+
+cd amplify-app
+```
+
+<BlockSwitcher>
+
+<Block name="Angular 15">
+
+First, create `src/polyfills.ts` and add the following:
+
+```javascript
+(window as any).global = window;
+(window as any).process = {
+  env: { DEBUG: undefined },
+};
+```
+
+Then, open your `angular.json` file, and add `src/polyfills.ts` to polyfills array(s) in your `angular.json`. These arrays are located in projects.`<project-name>.architect.<task-name>.options`.
+
+```javascript
+"polyfills": [
+  "zone.js",
+  "src/polyfills.ts"
+],
+```
+
+And finally, make sure to add `src/polyfills` to files in your `tsconfig.app.json`:
+
+```javascript
+{
+  "files": [
+    "src/main.ts",
+    "src/polyfills.ts"
+  ],
+}
+```
+</Block>
+
+<Block name="Angular 10-14">
+
+Add the following to your `src/polyfills.ts` file to recreate them: 
+
+```javascript
+(window as any).global = window;
+(window as any).process = {
+  env: { DEBUG: undefined },
+};
+```
+
+</Block>
+
+</BlockSwitcher>
 
 </Block>
 
 <Block name="ReactJS">
 
-<Fragments fragments={{ js: js2 }} />
+To get started, first create a new React app using Vite, and then install and use the Amplify CLI to start adding backend capabilities to your app.
+
+From your projects directory, run the following commands and follow the prompts:
+
+```bash
+npm create vite@latest
+```
+This creates a new React app in a new directory. Navigate to your new directory and run the app by using the following command:
+
+```bash
+cd <your project directory>
+```
+
+Then, install your dependencies and run the development server by running the following command:
+
+```bash
+npm install
+npm run dev
+```
+
+This runs a development server and see preview your app locally.
 
 </Block>
 
 <Block name="VueJS">
 
-<Fragments fragments={{ js: js3 }} />
+Use the Vue Vite powered `create-app` to bootstrap a new Vue 3 app (selecting the defaults will work for this project):
```

Comment:
```suggestion
Use the Vue Vite powered `create-app` to bootstrap a new Vue 3 app (select required configuration, selecting the defaults as an example for this project):
```

Reply:
nit, please feel free to ignore if it doesn't make sense

Reply:
Committed the change!